data/phones/lib/generate_phones_summary.py

```diff
@@ -68,34 +52,28 @@ def main() -> None:
             )
         iso639_code = file_path[: file_path.index("_")]
         if "broad" in file_path:
-            transcription_level = file_path[
-                file_path.index("broad") : file_path.index(".")
-            ].capitalize()
+            transcription_level = "Broad"
         else:
-            transcription_level = file_path[
-                file_path.index("narrow") : file_path.index(".")
-            ].capitalize()
-        wiki_name = _handle_wiki_name(
-            languages[iso639_code], file_path, modifiers
-        )
+            transcription_level = "Narrow"
+        wiki_name = _handle_wiki_name(languages[iso639_code], file_path)
         row = [
             iso639_code,
             languages[iso639_code]["iso639_name"],
             wiki_name,
             transcription_level,
             num_of_entries,
         ]
-        languages_summary_list.append([file_path] + row)
-        readme_list.append([f"[phone]({file_path})"] + row)
+        phones_summary_list.append([file_path] + row)
```

Comment:
drive-by, but `phones_summaries` is probably a better variable name than `phones_summary_list`.